IDE/Android/README.md

```diff
@@ -0,0 +1,43 @@
+Installing WolfProvider on Android
+
+# Prerequisites
+You'll need to get the [Android NDK](https://developer.android.com/ndk/downloads/). I used [this one](https://dl.google.com/android/repository/android-ndk-r26b-linux.zip). Extract it into the folder that contains all the other files.
```

Comment:
"into the folder that contains all the other files" needs to be more descriptive. Is this being extracted in wolfProvider root directory or adjacent to wolfProvider directory?

Reply:
Ahh I see, build.sh takes care of this step. Mentioning that would be good here.